packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -604,10 +604,10 @@ void main() {
     final DecoratedBox decoratedBox = tester.widgetList(find.descendant(
       of: find.byType(CupertinoNavigationBar),
       matching: find.byType(DecoratedBox),
-    )).first;
+    )).first as DecoratedBox;
```

Comment:
Same thing here - this actually could also just be `tester.firstWidget<DecoratedBox>` and drop the `.first`, but we don't necessarily have to change that with this patch.

Reply:
ditto